uint16 radius
---
ExtremaRegion min 
ExtremaRegion max
